ad7768-1fmcz/spi_engine.c

```diff
@@ -102,6 +102,63 @@ int32_t spi_eng_dma_read(spi_dev *dev,
 	return 0;
 }
 
+/***************************************************************************//**
```

Comment:
is there a common `spi_engine.c` file that could be used instead ?

Reply:
Not yet, but the latest and best version that we have so far is part of the [ad400x ](https://github.com/analogdevicesinc/no-OS/blob/master/ad400x-fmcz/spi_engine.c)project. We can probably use it in our future projects.